### DISCUSSION TITLE
Sample Avalonia App for GitHub Pages?

### DISCUSSION THREAD

**Discussion by rccarlson** [AvaloniaUI/Avalonia#15514] [Q&A] [answered]
Does anyone have an example of an Avalonia app running on GitHub Pages that I could use as a reference implementation?

I tried to publish [the demo app](https://docs.avaloniaui.net/docs/0.10.x/tutorials/running-in-the-browser) on GitHub Pages, but I'm getting errors that I can't figure out.

[Project](https://github.com/rccarlson/WebTest) and [Published GitHub Page](https://rccarlson.github.io/WebTest/)

![image](https://github.com/AvaloniaUI/Avalonia/assets/43221493/8723dc73-def8-4707-960f-4352657be681)

The error:
```
Failed to find a valid digest in the 'integrity' attribute for resource 'https://rccarlson.github.io/WebTest/_framework/dotnet.native.js.symbols' with computed SHA-256 integrity 'n2H5RwYCZw3slBzpDtDJJ3ZUN3vSMvu8njYveYkK6qM='. The resource has been blocked.
```

Clearly the `dotnet.native.js.symbols` file is failing an integrity check, but I don't know what I'm doing wrong that it's freaking out about it.

**Comment by rabbitism** [comment]
maybe you can check mine. 
https://github.com/irihitech/Semi.Avalonia

**Comment by rccarlson** [comment, marked as answer]
So after losing no small part of my sanity, I think this was git's fault.

When downloading the `dotnet.native.js.symbols` file directly from GitHub, the hash was indeed different, though git said they were the same file.

My solution was to go through the GitHub web interface, delete the `dotnet.native.js.symbols`, then upload the corrected copy from my computer. The website now works and correctly displays the Avalonia "hello world" application
